src/compiler/factory/nodeFactory.ts

```diff
@@ -1150,63 +1152,50 @@ export function createNodeFactory(flags: NodeFactoryFlags, baseFactory: BaseNode
     // Identifiers
     //
 
-    function createBaseIdentifier(escapedText: __String, originalKeywordKind: SyntaxKind | undefined) {
+    function createBaseIdentifier(escapedText: __String) {
         const node = baseFactory.createBaseIdentifierNode(SyntaxKind.Identifier) as Mutable<Identifier>;
-        node.originalKeywordKind = originalKeywordKind;
         node.escapedText = escapedText;
-        node.autoGenerate = undefined;
-        node.typeArguments = undefined;
-        node.hasExtendedUnicodeEscape = undefined;
         node.jsDoc = undefined; // initialized by parser (JsDocContainer)
-        node.jsDocCache = undefined; // initialized by parser (JsDocContainer)
         node.flowNode = undefined; // initialized by binder (FlowContainer)
         node.symbol = undefined!; // initialized by checker
         return node;
     }
 
     function createBaseGeneratedIdentifier(text: string, autoGenerateFlags: GeneratedIdentifierFlags, prefix: string | GeneratedNamePart | undefined, suffix: string | undefined) {
-        const node = createBaseIdentifier(escapeLeadingUnderscores(text), /*originalKeywordKind*/ undefined) as Mutable<GeneratedIdentifier>;
-        node.autoGenerate = {
+        const node = createBaseIdentifier(escapeLeadingUnderscores(text)) as Mutable<GeneratedIdentifier>;
+        setIdentifierAutoGenerate(node, {
             flags: autoGenerateFlags,
             id: nextAutoGenerateId,
             prefix,
             suffix
-        };
+        });

```

Comment:
Nit: indention here seems weird, and in other places below.

Reply:
Indentation seems fine to me?

Reply:
Oops, sorry, my brain merged the diff into a single piece of code.